src/Containers/WeatherToday/weatherToday.tsx

```diff
@@ -0,0 +1,132 @@
+import React, {useEffect, useState} from "react";
+import axios from "axios";
+import {WeatherCard} from "./components/WeatherCard/WeatherCard";
+import {SearchBar} from "./components/searchBar/searchBar";
+import {PreloaderComponent} from "../../Component/preloader/preloader";
+
+export const WeatherToday = (): JSX.Element => {
+    const [cityTitleValue, setCityTitleValue] = useState<string>('')
+    const [cityTitle, setCityTitle] = useState<string>('')
+    const [sunrise, setSunrise] = useState<string>('')
+    const [sunset, setSunset] = useState<string>('')
+    const [weatherState, setWeatherState] = useState<object>({
+        cityTitle: '',
+        humidity: '',
+        feelsLike: 0,
+        temperature: 0,
+        pressure: 0,
+})
+    const [isLoading, setIsLoading] = useState<boolean>(false)
+    const apiKEY = '362efd6a69303a0f5662a19ce25e5e89'
+    let filterTimeout: NodeJS.Timeout;
+
+    useEffect(() => {
+        setIsLoading(true)
+        getCurrentPosition()
```

Comment:
Somewhere you add ;, somewhere you don't. Either add ; everywhere, or don't add them at all